### ISSUE TITLE
Github Pages docs shouldn't require local Jekyll installation

### ISSUE THREAD

**Issue by hansfn** [github/docs#9066]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's Code of Conduct.

### What article on docs.github.com is affected?

https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll

### What part(s) of the article would you like to see updated?

Under "Prerequisites" it currently reads:

> Before you can use Jekyll to create a GitHub Pages site, you must install Jekyll and Git.

This makes 100 % sense if you want to test / build locally. However, if you are not testing locally, it is unclear to me why you require Jekyll. I can only imagine the reason is that you use `jekyll new` to create a template / starter site. I find the creation of a template site using Jekyll itself unnecessary. For most people it's enough to select a theme and add pages/posts. (And reading the Jekyll / theme documentation about how to achieve certain things, of course.) Some themes even have template project themselves.

My main point is that if you select Github Pages with the included Jekyll site generator, you do it for simplicity. Requiring a local installation of Jekyll is contradictory to that. And it's not needed as the page is built on Github servers. 

I suggest that you make it clear that you can use Github Pages with Jekyll without any local installation.

**Comment by ramyaparimi** [comment]
@hansfn Thanks so much for opening an issue! I'll triage this for the team to take a look :eyes:

**Comment by lecoursen** [comment]
>My main point is that if you select Github Pages with the included Jekyll site generator, you do it for simplicity. Requiring a local installation of Jekyll is contradictory to that. And it's not needed as the page is built on Github servers.

Folks interested in a simpler version of creating a GitHub Pages site can follow the instructions in "[Creating a GitHub Pages site](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-your-site)" instead. "Creating a GitHub Pages site with Jekyll" is intended for an audience that does want the benefits of a more complex local environment. Since installing Jekyll is a prerequisite for the procedure presented in the article, we'll leave it there. Thanks for your interest in contributing to our docs!

**Comment by hansfn** [comment]
Thank you for responding. 

**TLDR**: Your documentation is excellent for people who want to test their site locally, but it should be made 100% clear that no local installation is needed to get the full power of Jekyll - and use it to "customize your site even more".

A lot of us developers are lazy and enjoy Github Pages because you can get the **full** power of Jekyll without any effort - just create a Jekyll site and push it. No local Jekyll installation or CI / Github Actions is needed. This is the simplicity I talk about. And from other Github Pages issues I can tell that I'm not alone loving this.

Running `jekyll new --skip-bundle .` (step 7) does create a very, very simple example site, that you as well can create by just reading the Jekyll documentation. That is the only thing you gain from all the steps (and potential errors) - unless you actually plan to test the site locally. Githib Pages doesn't even use the `Gemfile.lock` generated by `bundle`.

If you read the "[Next steps](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#next-steps)" in "Creating a GitHub Pages site" documentation, you'll find:

> To customize your site even more, you can use Jekyll, a static site generator with built-in support for GitHub Pages. For more information, see "About GitHub Pages and Jekyll."

Which tells you to install Jekyll locally - which is 100% unnecessary to "customize your site even more".
